static/bumble_remix.js

```diff
@@ -0,0 +1 @@
;\n  }\n}\n\nfn isInTriangle(a: vec2f, b: vec2f, c: vec2f, p: vec2f) -> bool {\n  return ( \n    (c.x - p.x) * (a.y - p.y) - (a.x - p.x) * (c.y - p.y) >= 0 &&\n    (a.x - p.x) * (b.y - p.y) - (b.x - p.x) * (a.y - p.y) >= 0 &&\n    (b.x - p.x) * (c.y - p.y) - (c.x - p.x) * (b.y - p.y) >= 0\n  );\n}\n\nfn area(a: vec2f, b: vec2f, c: vec2f) -> f32 {\n  return (b.y - a.y) * (c.x - b.x) - (b.x - a.x) * (c.y - b.y);\n}"}},726:(e,t,r)=>{r.d(t,{A:()=>n});const n={code:"const NUM_DROPS: u32 = #NUM_DROPS;\nconst NUM_DROP_VERTICES: u32 = #NUM_DROP_VERTICES;\nconst TRIANGLES_GENERATED: u32 = #TRIANGLES_GENERATED;\n\nstruct DropUniforms {\n  colors: array<vec3f, NUM_DROPS>,\n  currentDrop: f32,\n  aspectRatio: vec2f,\n  xyr: vec3f,\n}\n\n@group(0) @binding(0) var<storage> vertices: array<vec2f>;\n@group(0) @binding(1) var<storage, read_write> output_vertices: array<vec2f>;\n@group(0) @binding(2) var<uniform> drops: DropUniforms;\n\n@compute\n@workgroup_size(#SIMULATE_WORKGROUP_SIZE)\nfn computeMain(@builtin(global_invocation_id) drop: vec3u) {\n  let vertexIndex = drop.x;\n  if (vertexIndex >= NUM_DROPS * NUM_DROP_VERTICES) {\n    return;\n  }\n  let c = drops.xyr.xy;\n  let r = drops.xyr.z;\n  let dropIndex = vertexIndex / NUM_DROP_VERTICES;\n  if (dropIndex != u32(drops.currentDrop)) {\n    // // normal case\n    let v = vertices[vertexIndex];\n    let pMinusC = (v.x-c.x)*(v.x-c.x)+(v.y-c.y)*(v.y-c.y);\n    let lastTerm = sqrt(1 + r*r / pMinusC);\n    output_vertices[vertexIndex] = c + (v - c) * lastTerm;\n  } else {\n    // init case\n    let i = f32(vertexIndex % NUM_DROP_VERTICES);\n    let angle = i * ((radians(180) * 2) / f32(NUM_DROP_VERTICES));\n    output_vertices[vertexIndex] = vec2f(cos(angle) * r + c.x, sin(angle) * r + c.y);\n  }\n}"}}},i={};function o(e){var t=i[e];if(void 0!==t)return t.exports;var r=i[e]={exports:{}};return a[e](r,r.exports,o),r.exports}e="function"==typeof Symbol?Symbol("webpack queues"):"__webpack_queues__",t="function"==typeof Symbol?Symbol("webpack exports"):"__webpack_exports__",r="function"==typeof Symbol?Symbol("webpack error"):"__webpack_error__",n=e=>{e&&e.d<1&&(e.d=1,e.forEach((e=>e.r--)),e.forEach((e=>e.r--?e.r++:e())))},o.a=(a,i,o)=>{var s;o&&((s=[]).d=-1);var u,d,c,l=new Set,p=a.exports,f=new Promise(((e,t)=>{c=t,d=e}));f[t]=p,f[e]=e=>(s&&e(s),l.forEach(e),f.catch((e=>{}))),a.exports=f,i((a=>{var i;u=(a=>a.map((a=>{if(null!==a&&"object"==typeof a){if(a[e])return a;if(a.then){var i=[];i.d=0,a.then((e=>{o[t]=e,n(i)}),(e=>{o[r]=e,n(i)}));var o={};return o[e]=e=>e(i),o}}var s={};return s[e]=e=>{},s[t]=a,s})))(a);var o=()=>u.map((e=>{if(e[r])throw e[r];return e[t]})),d=new Promise((t=>{(i=()=>t(o)).r=0;var r=e=>e!==s&&!l.has(e)&&(l.add(e),e&&!e.d&&(i.r++,e.push(i)));u.map((t=>t[e](r)))}));return i.r?d:o()}),(e=>(e?c(f[r]=e):d(p),n(s)))),s&&s.d<0&&(s.d=0)},o.d=(e,t)=>{for(var r in t)o.o(t,r)&&!o.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},o.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),o(156)})();
```

Comment:
oof yeah ok I don't love just having this obfuscated/webpacked output turd from another build alongside the source here....might be nice sometime to figure out a way to publish this from the [other project](https://github.com/bzztbomb/webgpu_marbling/) or to add a build step here that does this based on the source in the other repo? I'll open an issue.